### ISSUE TITLE
CSS is behaving strangely.

### ISSUE THREAD

**Issue by natGalynker** [ga-wdi-boston/game-project#213]
When I set the boardgame's div element to 600px for width and height and each tile to 1/3 of both the width and height the CSS was not communicating with the HTML. The browser and dev tools showed that instead of the elements I set properties and values to, that none of that was loading because for some reason it thought that I set some properties to the entire body of the the html

`````` html
<body>
      <h1>Example</h1>
      <form id="sign-up">
        <fieldset>
          <legend>Sign up!</legend>
          <input type="text" name="credentials[player_name"
          placeholder="player name">
          <input type="text" name="credentials[email]" placeholder="e-mail">
          <input type="password" name="credentials[password]"
          placeholder="password">
          <input type="password" name="credentials[password_confirmation]"
          placeholder="confirm password">
          <input type="submit" name="submit" value="Sign up!">
        </fieldset>
      </form>
      <form id="sign-in">
        <fieldset>
          <legend>Sign in!</legend>
          <input type="text" name="credentials[email]" placeholder="e-mail">
          <input type="password" name="credentials[password]"
          placeholder="password">
          <input type="text" name="credentials[player_name]"
          placeholder="player">
          <input type="submit" name="submit" value="Sign in!">
        </fieldset>
      </form>
      <form id="change-password">
        <fieldset>
          <legend>Change password!</legend>
          <input type="password" name="passwords[old]"
          placeholder="old password">
          <input type="password" name="passwords[new]"placeholder="new password">
          <input type="submit" name="submit" value="Change Password!">
        </fieldset>
      </form>
      <form id="sign-out">
        <fieldset>
          <legend>Log Out!</legend>
          <input type="password" name="passwords[old]"
          placeholder="old password">
          <input type="password" name="passwords[new]"placeholder="new password">
          <input type="submit" name="submit" value="Logout!">
        </fieldset>
      </form>
      <form id="change-player">
        <fieldset>
          <legend>Change Player!</legend>
          <input type="text" name="player_name[old]"
          placeholder="new player">
          <input type="text" name="player_name[new]" placeholder="new player">
          <input type="password" name="credentials[password]"
          placeholder="old password">
          <input type="submit" name="submit" value="Update!">
        </fieldset>
      </form>
      <div class="main">
        <div class="box-1 space">
          <p>Test</p>
        </div>
        <div class="box-2 space">
          <p>Test</p>
        </div>
        <div class="box-3 space">
          <p>Test</p>
        </div>
      </div>
      <div class="box-4 space">
        <p>Test</p>
      </div>
      <div class="box-5 space">
        <p>Test</p>
      </div>
      <div class="box-6 space">
        <p>Test</p>
      </div>
      <div class="box-7 space">
        <p>Test</p>
      </div>
      <div class="box-8 space">
        <p>Test</p>
      </div>
      <div class="box-9 space">
        <p>Test</p>
      </div>
    </body>```

.main {
  background-color: #6EA8A9;
  overflow: hidden;
  width: 300px;
  height:300px;
}

.space {
  height: 100px;
  width: 100px;
}
``````

**Comment by laurenfazah** [comment]
Sorry, I'm not sure I'm understanding the issue. Could you rephrase?

**Comment by natGalynker** [comment]
Yeah, I was struggling to find the proper words. When I attempted to resize the width and height of the boardgame div, so that each box would be 1/3 of the height and width. The following occurred:
all the boxes lined up either along the left side of the screen to the very bottom, or a few would be on the screen then the rest would be on the very bottom of the screen, which is supposed to be outside of the scope of the gameboard. Dev tools told me when I selected elements that it was the body div which it was loading, and I don't see any of my style rules showing up in the Dev tools
